builder/vmware/common/driver_esx5.go

```diff
@@ -679,17 +679,28 @@ func (d *ESX5Driver) Download(src, dst string) error {
 	return d.comm.Download(d.datastorePath(src), file)
 }
 
-func (d *ESX5Driver) VerifyChecksum(ctype string, hash string, file string) bool {
-	if ctype == "none" {
+func (d *ESX5Driver) VerifyChecksum(hash string, file string) bool {
+	if hash == "none" {
 		if err := d.sh("stat", strconv.Quote(file)); err != nil {
 			return false
 		}
-	} else {
-		stdin := bytes.NewBufferString(fmt.Sprintf("%s  %s", hash, file))
-		_, err := d.run(stdin, fmt.Sprintf("%ssum", ctype), "-c")
-		if err != nil {
-			return false
-		}
+		return true
+	}
+
+	// TODO(azr): this will not work with checksums from files
```

Comment:
Here I need to use what's in https://github.com/hashicorp/packer/pull/9129

Reply:
I think you definitely need to complete this TODO before this can be considered ready to merge.

Reply:
💯 !!